src/schema/fair.ts

```diff
@@ -47,6 +49,44 @@ const FairType = new GraphQLObjectType({
   name: "Fair",
   fields: () => ({
     ...GravityIDFields,
+    artistsConnection: {
+      type: artistConnection,
+      args: pageable({
+        section: {
+          type: GraphQLString,
+          description: "Number of artists to return",
+        },
+        sort: {
+          type: FairArtistSortsType,
+          description: "Sorts for artists in a fair",
+        },
+      }),
+      resolve: (
+        { id },
+        options,
+        _request,
+        { rootValue: { fairArtistsLoader, artistsLoader } }
+      ) => {
+        const gravityOptions = omit(
+          convertConnectionArgsToGravityArgs(options),
+          ["page"]
+        )
+
+        return Promise.all([
+          totalViaLoader(fairArtistsLoader, id, gravityOptions),
```

Comment:
You might be able to drop this call, and add `gravityOptions.total_count = true` somewhere. Then, you can resolve `headers` as well as `body` of `fairArtistsLoader`, and retrieve the count in `headers['x-total=count']`.